apps/web/src/pages/tenants/components/list/columns.tsx

```diff
@@ -0,0 +1,42 @@
+import { ITenantEntity } from '@novu/shared';
+import { Skeleton } from '@mantine/core';
+import { format } from 'date-fns';
+
+import { IExtendedColumn } from '../../../../design-system';
+
+const maxWidth = 60;
+
+export const columns: IExtendedColumn<ITenantEntity>[] = [
+  {
+    accessor: 'name',
+    Header: 'Name',
+  },
+  {
+    accessor: 'identifier',
+    Header: 'Tenant identifier',
+  },
+  {
+    accessor: 'createdAt',
+    Header: 'Create at',
```

Comment:
```suggestion
    Header: 'Created At',
```

Reply:
Make it like this because of the design 🙃

Reply:
Then just create -> created

Reply:
Oh sorry you're right! 
it should be `Created at` missed the past part